Release v1.0.0

aldemeery opened the pull request:
### Added

-   HTML structure for the entire page.
-   Styling for the header and logo.
-   Style the form.
-   Style the footer.
-   Style the privacy settings button.

aldemeery commented:
> Important 🔴 Please enable you stickler configuration, they represent style guidelines that you share with your team - which is a group of all Microverse students

Already enabled.

>  Make sure your HTML code passes all validations, you can do it by direct input

The code already passes the W3Validator, there were no errors, it was only a warning, but even that is removed now.

> The form should contain several icons with a intuit logo. But you have tried to cover it using a screenshot. So please add separate icons with their respective name.

Not sure what is the problem here, we already have three separate icons, not screenshots, please elaborate.

> Also make sure they are in the same order as the original one (turbolink, quickbooks and mint).

Actually they can be in a different order depending on where you open the link from, so for the task, we were required to mimic the registration form of [mint.com](https://mint.com), and if you go there and then click `sign up` you will get the form that we did mimic, and you can see our order is correct..please check it out [here](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint)

> Take a moment to compare the font family, color and size of your clone with the original page. In most cases, it seems yours have a darker color and bigger size, make sure they look as much as possible the same.

Updated

> Please update the live demo link

Updated

rammazzoti2000 commented:
For product icons please do not use screenshots, you should use icons and text, just like in the original. Stickler is configured but not enabled read again the guide please on their [repo](https://github.com/microverseinc/linters-config/tree/master/css) and make sure to enable it https://stickler-ci.com/.
The code must pass all validations including warnings.

aldemeery commented:
> For product icons please do not use screenshots, you should use icons and text, just like in the original.

Updated

> Stickler is configured but not enabled read again the guide please on their repo and make sure to enable it https://stickler-ci.com/.

Enabled and fixed all errors.
